userguide/glossary/Kill-Bill-Glossary.adoc

```diff
@@ -95,6 +104,28 @@ Fixed Term:: A type of <<Phase,phase>> that specifies an end date and during whi
 
 == I
 
+Invoice:: An invoice is a bill provided to a customer for charges that are payable on the customer's account. A single invoice consolidates charges for all subscriptions held by that customer for a specified time period. Invoices may be set to be paid automatically, or may be paid manually by the customer. An invoice consists of one or more invoice items.
+
+Invoice Item:: Represents a single item charged on an invoice. Given an active subscription, one could see multiple items for that subscription on a single invoice including recurring items, usage items, fixed price items, etc. There can also be items for different subscriptions on the same invoice, as well as items that are unrelated to subscriptions such as adjustments and taxes. 
+
+
+Invoice Item Type:: Specifies the type of invoice item. Can be one of `EXTERNAL_CHARGE`, `FIXED`, `RECURRING`, `REPAIR_ADJ`, `CBA_ADJ`, `ITEM_ADJ`, `USAGE`, `TAX`, `PARENT_SUMMARY`. Refer to the
+https://docs.killbill.io/latest/userguide_subscription.html#components-invoicing[_Subscription Billing_] document to know more. 
+
+Invoice Payment:: Payment made against a specific invoice. By default, Kill Bill attempts to make a payment against an invoice using the account's default payment method. 
+
+Invoice Refund:: Refund amount associated with an invoice. Kill Bill allows issuing a full or partial refund against an invoice payment. 
```

Comment:
I think the following would make more sense: `Refund amount associated with an Invoice Payment.` Refunds are against payment, and the terminology `Invoice Refund` really means `(Invoice Payment) Refund`. Happy to hear other suggestions...

Reply:
Changed to `Refund amount associated with an invoice payment`. If anyone has any other suggestions, I can make changes accordingly.

Reply:
That sounds more accurate.

Reply:
@FlowWriter, are you simply agreeing with @sbrossie here or is there an action item for me? 😀